tutorials/ui/bbcode_in_richtextlabel.rst

```diff
@@ -931,10 +932,37 @@ Ghost
         char_fx.color.a = alpha
         return true
 
+ .. code-tab:: csharp
+ 
+    #if TOOLS
+    using Godot;
+
+    [Tool]
+    public class RichTextGhost : RichTextEffect
+    {
+        // Syntax: [ghost freq=5.0 span=10.0][/ghost]
+
+        // Define the tag name.
+        public string bbcode = "ghost";
```

Comment:
Ah, I see. It looks like this only works with scripts that have a property named exactly `bbcode`:

https://github.com/godotengine/godot/blob/be56cab58c056c074d1e02cd0b38641204e39f41/scene/gui/rich_text_effect.cpp#L49

That's a weird pattern, I don't think we usually do that. For example, in editor plugins there are virtual methods for doing this kind of thing (e.g.: `EditorPlugin::_get_plugin_name`).

So to make this work the member would have to be named `bbcode` even though that goes against our C# naming conventions. And it can't be a constant.